guides/v2.1/cloud/cdn/configure-fastly.md

```diff
@@ -150,19 +150,24 @@ You cannot use the Fastly service in Integration environments.
 2.	Wait for dependencies to be updated.
 3.	Enter the following command to fully update and clear the cache:
 
-		php bin/magento setup:upgrade && php bin/magento cache:clean
+```bash
+php bin/magento setup:upgrade && php bin/magento cache:clean
```

Comment:
The convention is just `bin/magento command...`
You can remove the `php` from all these commands.

Reply:
> The convention is just bin/magento command...
> You can remove the php from all these commands.

**implemented**

Reply:
The blanket PHP changes made can not apply to Cloud documentation. Some of those affect sample code for copy to YAML files and will cause errors without the "PHP". Please revert changes to Cloud documentation for PHP command.

Reply:
> The blanket PHP changes made can not apply to Cloud documentation. Some of those affect sample code for copy to YAML files and will cause errors without the "PHP". Please revert changes to Cloud documentation for PHP command.

reverted for Cloud documentation